lib/configs/react.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Maybe I should distinguish this config as being accessibility rules for react....like `github/jsx-a11y`. 

thoughts? @github/accessibility-reviewers

Reply:
I think for future rules github/react is kinda nice?

Reply:
Yeah I'm not sure either... I'm down to keep this for now and revisit later!